app/services/companies_details_updator.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+class CompaniesDetailsUpdator < ApplicationService
+  def call
```

Comment:
I would split it into 2 methods:

```suggestion
  def call
    update_details_of_gpw_companies
    update_details_of_nc_companies
  end
```